File: manga_be/src/main/java/com/reader/manga/adapters/input/dtos/animes/AnimeDTO.java
package com.reader.manga.adapters.input.dtos.animes;

import lombok.Builder;
import jakarta.validation.constraints.NotBlank;
import jakarta.validation.constraints.NotNull;
import org.springframework.web.multipart.MultipartFile;

@Builder
public record AnimeDTO(
        MultipartFile file,
        @NotNull @NotBlank String id,
        @NotNull @NotBlank String title
) {}

File: manga_fe/src/class/DTO/AnimeDTO.ts
class AnimeDTO {
    readonly file: File;
    readonly id: string;
    readonly title: string;

    constructor(file: File, id: string, title: string) {
        this.id = id;
        this.file = file;
        this.title = title;
    }
}

export default AnimeDTO;